pydevtools/profiling_late.py

```diff
@@ -46,14 +50,14 @@ def declare_timepoint(name, isStart):
                 pythonProfile = True
                 if isStart:
                     if point_current:
-                        logger.error('Stop profiling for point "{0}" and do "{1}" instead'.format(point_current, name))
+                        print 'Stop profiling for point "{0}" and do "{1}" instead'.format(point_current, name)
```

Comment:
Small thing here: as Python 2 has an expiration date, maybe it would be a good idea to use the print format from Python 3. Other than that 👍